pkg/question/question.go

```diff
@@ -642,7 +687,9 @@ func AskVpc(h *ec2helper.EC2Helper) (*string, error) {
 				vpcName = fmt.Sprintf("%s(%s)", *vpcTagName, *vpc.VpcId)
 			}
 
-			if *vpc.IsDefault {
+			if userDefaultVpcId != nil && *vpc.VpcId == *userDefaultVpcId {
+				defaultOptionRepr, defaultOptionValue = vpcName, *vpc.VpcId
+			} else if *vpc.IsDefault && defaultOptionValue == cli.ResponseNew {
 				defaultOptionRepr, defaultOptionValue = vpcName, *vpc.VpcId
 			}
```

Comment:
Combine the conditions
```suggestion
			if (userDefaultVpcId != nil && *vpc.VpcId == *userDefaultVpcId) ||
			   (*vpc.IsDefault && defaultOptionValue == cli.ResponseNew) {
				defaultOptionRepr, defaultOptionValue = vpcName, *vpc.VpcId
			}
```